distributions/nuxt-ts/bin/nuxt-ts.js

```diff
@@ -1,5 +1,46 @@
 #!/usr/bin/env node
 
+const boxen = require('boxen').default
+const chalk = require('chalk').default
+
+const warningBox = boxen([
+  chalk.yellow.bold('IMPORTANT : Package deprecation'),
+  '',
+  `Nuxt TypeScript Support has been refactored to be used with ${chalk.green.bold('nuxt')} package.`,
+  `Which means that ${chalk.yellow.bold(`nuxt-ts`)} package is now longer needed and is now tagged as ${chalk.yellow.bold('deprecated')}.`,
```

Comment:
should be "nuxt-ts package is now no longer needed"

Reply:
Yes, thanks @Niputi !